orchestra/google_apps/convenience.py

```diff
@@ -217,6 +223,16 @@ def upload_file(parent_id, file_path, title, description, mimetype):
         description,
         parent_id,
         mimetype,
+        mimetype,
         file_path
     )
     return file_metadata
+
+
+def get_google_spreadsheet_as_csv(gdoc_url, reader=csv.DictReader):
+    matches = GSPREAD_RE.match(gdoc_url)
+    assert matches is not None
```

Comment:
raise a nicer error? (This is what shows up in the interface if you typo a URL or try to import a non-google-spreadsheets URL, right?)